packages/messaging/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Unreleased
+- [fixed] Fixed an issue introduced in 7.7.0 where the FCM switched to provide base64-encoded VAPID key
```

Comment:
"Fixed an issue where" is a phrase often followed by what the issue is.  

I think you may mean "an issue introduced in 7.7.0, when FCM switched to providing base64-encoded VAPID keys..."